constants/minimum.investment.js

```diff
@@ -0,0 +1,7 @@
+const MinimumInvestment = {
```

Comment:
Why is there different minimum investments for each portfolio?

Reply:
Because if your portfolio has, i.e 0.1 ETH, you need to make sure that 0.1 * WhateverAmount is at least the minimum required by Buda. So, having that value as a constant avoids trouble

Reply:
After giving it some thought maybe it's simpler to just have a MinimumInvestment that works for all Portfolios? In this case it would be the Max value of all three.